scene/resources/portable_compressed_texture.cpp

```diff
@@ -30,6 +30,8 @@
 
 #include "portable_compressed_texture.h"
 
+#include "compressed_texture.h"
+#include "core/config/project_settings.h"
 #include "core/io/marshalls.h"
 #include "scene/resources/bit_map.h"
```

Comment:
We use absolute paths even when in the same folder (aside from the .h file matching the .cpp).
```suggestion
#include "core/config/project_settings.h"
#include "core/io/marshalls.h"
#include "scene/resources/bit_map.h"
#include "scene/resources/compressed_texture.h"
```